packages/matchbox/src/components/Toggle/styles.js

```diff
@@ -0,0 +1,68 @@
+import { tokens } from '@sparkpost/design-tokens';
+import styled from 'styled-components';
+
+export const toggle = props => `
+  display: inline-block;
+  position: relative;
+  height: 1.25rem;
+  width: 2.25rem;
+
+  opacity: ${props.disabled ? '0.6' : '1'}
+
+  &:hover {
+    cursor: ${props.disabled ? 'not-allowed' : 'pointer'};
+  }
+`;
+
+export const indicator = props => `
+  pointer-events: none;
+  position: absolute;
+  top: 1px;
+  left: 1px;
+  bottom: 1px;
+  width: calc(2.25rem / 2);
+
+  background: ${tokens.color_white};
+  transition: ${tokens.motionDuration_fast} ${tokens.motionEase_in_out};
+  transform: translate(${props.checked ? tokens.spacing_400 : '0'}, 0);
+
+  border-radius: ${tokens.borderRadius_circle};
+  box-shadow: ${tokens.boxShadow_100};
+`;
+
+export const outline = () => `
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
+  background: ${tokens.color_gray_600};
+  border-radius: ${tokens.borderRadius_pill};
+  transition: ${tokens.motionDuration_fast};
+`;
+
+export const StyledIndicator = styled('span')`
+  ${indicator}
+`;
+
+export const StyledOutline = styled('span')`
+  ${outline}
+`;
+
+export const input = () => `
+  &:focus ~ ${StyledOutline},
+  &:active ~ ${StyledOutline} {
+    box-shadow: 0 0 0 2px white, 0 0 0 4px ${tokens.color_blue_600};
+  }
+
+  &:checked {
+    & ~ ${StyledIndicator} {
+      transform: translate(${tokens.spacing_400}, 0);
+    }
+
+    & ~ ${StyledOutline} {
+      background: ${tokens.color_green_600};
```

Comment:
This is a slightly unusual situation, but unfortunately this isn't meeting color contrast guidelines - `#fff` on `#28bd8b` doesn't pass.

There are a couple of ways to handle this:
1. Darken the green a lot
2. Make the box shadow on the switch a good bit darker, or add an outline to the white switch element that meets the criteria

Then technically, if the darkest part of the box shadow passes against `#28bd8b` then you're good. Had a very similar situation with a very similar component a few months ago.